docs/source/release_notes/release-notes.rst

```diff
@@ -63,7 +63,11 @@ develop
            This has empirically shown to be a better batch size because it puts less stress on the underlying KVS.
            For a full list of tunable sweep parameters and default settings, see :ref:`sweep tunable options <sweep_tunable_parameters>`.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1763>`__)
-       
+
+    *    - |improved|
```

Comment:
After #1779 merges, you'll need to rebase this, as this block will now appear in 0.38.0.
Release note looks OK, but could you also add a link to the docs? Something like "For more information, please consult the :ref:`documentation <...>`."